src/atoms/ButtonQR/ButtonQR.js

```diff
@@ -181,6 +190,11 @@ class ButtonQR extends React.PureComponent<Props> {
 							<CheckSVG />
 						</CompleteCover>
 					)}
+					{isExpired && (
+						<ExpiredCover>
+							<XSVG />
+						</ExpiredCover>
+					)}
```

Comment:
Update the ButtonQR story to contain this `expired` state

Reply:
6a53e9e